pgmpy/base/DAG.py

```diff
@@ -1090,6 +1090,77 @@ def to_graphviz(self):
         """
         return nx.nx_agraph.to_agraph(self)
 
+    def fit(self, data, estimator=None, state_names=[], n_jobs=1, **kwargs):
```

Comment:
You would also need to delete this method from BayesianNetwork class.

Reply:
Should i completely remove it and any test cases if present.

Reply:
Yes, you should remove the method, otherwise it is just duplicated code. Test cases should stay as we do not expect any change in the behaviour of the BayesianNetwork class with these changes.

Reply:
Sure, understood